torch_audiomentations/__init__.py

```diff
@@ -1,6 +1,8 @@
 from .augmentations.polarity_inversion import PolarityInversion
 from .augmentations.impulse_response import ApplyImpulseResponse
+from .augmentations.background_noise import ApplyBackgroundNoise
 from .utils.convolution import convolve
 from .utils.file import find_audio_files, load_audio
+from .utils.dsp import calculate_rms, calculate_desired_noise_rms, resample_audio
```

Comment:
I would limit the functions imported in `__init__` to what will be used by the user. @iver56 ?

Reply:
Yes, that makes sense. I've recently done the same thing in audiomentations. If some users want to use our internal utils, they can do a workaround, like copying the code or doing a deeper import (and taking the risk that internals might change across versions).